### PR TITLE
enable ccache

### PR DESCRIPTION
**Pull request by jschleicher**
to speed up subsequent builds

### CODE REVIEW

**Review by ct2034 — APPROVED** [review]
this makes sense. I thought we did this already.

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ sudo: required
 dist: trusty
 language: generic
 
+cache:
+  directories:
+    - $HOME/.ccache
+
```
Comment (ct2034) [inline .travis.yml:8]:
```suggestion
cache: ccache
```
similar to https://github.com/PilzDE/pilz_industrial_motion/pull/254

**Review by ct2034 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (ct2034) [reply .travis.yml:8]: according to https://docs.travis-ci.com/user/caching/#ccache-cache

Comment (jschleicher) [reply .travis.yml:8]: @ct2034 the shorter form doesn't seem to work - see https://github.com/PilzDE/pilz_industrial_motion/pull/254#pullrequestreview-373562783

Comment (ct2034) [reply .travis.yml:8]: that seems odd.

**Review by ct2034 — APPROVED** [review]

**Pull request merged**